### DISCUSSION TITLE
Next.js support for Storefront VS Next.js commerce plugin update

### DISCUSSION THREAD

**Discussion by gorjan-mishevski** [saleor/saleor#7244] [Q&A]
Hi,
I was wondering about the Saleor support for Next.js integration. I've noticed the Storefront has a todo project for introducing Next 10, but also there is the draft PR for Next.js commerce integration.
What is the difference between them? Is Next.js commerce a better starter template combined with the upcoming Saleor integration?
Thank you

**Comment by krzysztofwolski** [comment]
Nextjs != nextjs commerce. The plan is to migrate our current storefront to use NextJS which would allow us to leverage SSR for better SEO.

Besides that, ATM Nexjs e-commerce and Vuestroefront integrations are in the making by external teams.
Differences are in the used tech stack and supported features. Since Saleor is a headless platform, it can be used with different storefronts, so you will be able to pick the one you like the most ;)

**Reply by gorjan-mishevski** [reply]
Thank you for your swift answer, one follow-up question, just of curiosity. How long do you think it will take for the next.js commerce integration and/or the Saleor storefront migration for Next.js to be done? :)
